### PR TITLE
[TW-342] Add liquidity baking improvement

### PR DESCRIPTION
**Pull request by MuKoJla**

### CODE REVIEW

File: src/hooks/use-sirs-token.hook.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { useTokenSelector } from '../store/wallet/wallet-selectors';
+import { LIQUIDITY_BAKING_LP_SLUG } from '../token/data/token-slugs';
```
Comment (unixvb) [inline src/hooks/use-sirs-token.hook.ts:2]: rename to `SIRS_SLUG`  (and related variables)

File: src/styles/colors.ts [hunk 2]
```diff
@@ -25,7 +26,8 @@ const basicDarkColors: typeof basicLightColors = {
   black: '#ffffff',
   white: '#000000',
   peach: '#FF5B00',
-  liteOrange: '#242424'
+  liteOrange: '#242424',
+  orangeTransparent: '#F6D5C3'
```
Comment (unixvb) [inline src/styles/colors.ts:30]: reuse real alpha color (like `basicLightAlphaColors`)  
and let designer fix this

File: src/store/wallet/wallet-selectors.ts [hunk 3]
```diff
@@ -87,6 +88,12 @@ export const useTokensListSelector = () => {
   return useMemo(() => assetsList.filter(({ artifactUri }) => !isDefined(artifactUri)), [assetsList]);
 };
 
+export const useTokenSelector = (tokenSlug: string) => {
+  const tokensList = useTokensListSelector();
+
+  return useMemo(() => tokensList.find(({ address, id }) => getTokenSlug({ address, id }) === tokenSlug), [tokensList]);
```
Comment (unixvb) [inline src/store/wallet/wallet-selectors.ts:94]: `tokenSlug` is missed in `deps`

File: src/hooks/use-sirs-token.hook.ts [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+import { isDefined } from '../utils/is-defined';
+
+export const useSirsToken = () => {
+  const sirsToken = useTokenSelector(LIQUIDITY_BAKING_LP_SLUG);
+
+  const isSirsBalance = isDefined(sirsToken) && Number(sirsToken.balance) > 0;
+
+  return { isSirsBalance, sirsToken };
+};
```
Comment (unixvb) [inline src/hooks/use-sirs-token.hook.ts:11]: rename hook and properties returned by the hook.  

what I don't like:  
1. useSirs**Token** do not return TokenInterface (mb could be renamed to useSirsInfo)
2. use**Sirs**Token returns { is**Sirs**Balance, **sirs**Token } (duplication)
3. isSirsBalance do not represent what it is actually (mb isPositiveBalance)

**Review by unixvb — DISMISSED** [review]

**Review by oryhivskiy — APPROVED** [review]

**Review by unixvb — APPROVED** [review]

**Pull request merged**